packages/launchpad/cypress/e2e/choose-a-browser.cy.ts

```diff
@@ -132,20 +134,36 @@ describe('Choose a Browser Page', () => {
       cy.findByRole('radio', { name: 'Chrome v1', checked: true }).as('chromeItem')
       cy.findByRole('radio', { name: 'Firefox v5', checked: false }).as('firefoxItem')
 
-      cy.contains('button', 'Launch Chrome').should('be.visible')
+      cy.contains('button', 'Start E2E testing in Chrome').should('be.visible')
 
       cy.intercept('mutation-OpenBrowserList_SetBrowser').as('setBrowser')
 
-      cy.get('@firefoxItem').then(($input) => {
-        cy.get(`label[for=${$input.attr('id')}]`).click().then(() => {
-          cy.wait('@setBrowser').its('request.body.variables.id').should('eq', $input.attr('id'))
-        })
+      cy.get('@firefoxItem').click().find('label').then(($label) => {
+        cy.wait('@setBrowser').its('request.body.variables.id').should('eq', $label.attr('for'))
       })
 
       cy.findByRole('radio', { name: 'Chrome v1', checked: false })
       cy.findByRole('radio', { name: 'Firefox v5', checked: true })
 
-      cy.contains('button', 'Launch Firefox').should('be.visible')
+      cy.contains('button', 'Start E2E testing in Firefox').should('be.visible')
```

Comment:
I would like to also add a test here to check that multiple clicks on the launch button don't send multiple mutations over. This would completely close UNIFY-465

Reply:
[packages/launchpad/cypress/e2e/choose-a-browser.cy.ts](https://github.com/cypress-io/cypress/blob/elevatebart/fix%2Fblock-double-click/packages/launchpad/cypress/e2e/choose-a-browser.cy.ts#L125-L165)

Reply:
It was still possible in my PR to launch two mutations via `cy.dbclick()`, so I lifted the logic from your PR and added it [here](https://github.com/cypress-io/cypress/pull/19830/commits/b7c02d94501ca769b30645003fa519663b23feba) and added some tests around it.

Reply:
grandiose !!!